cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -160,6 +160,7 @@ ExternalProject_Add(
     ${configure_command}
     ${configure_opts}
     "--prefix=@HUNTER_PACKAGE_INSTALL_PREFIX@"
+    "--libdir=lib"
```

Comment:
I'm just curious: did the default change or are you just explicit?

Reply:
The default changed specifically on amd64 linux to lib64, this keeps it in line to prevent excessive changes to `FindOpenSSL`